packages/flame/test/events/flame_game_mixins/has_draggable_components_test.dart

```diff
@@ -2,26 +2,28 @@ import 'package:flame/components.dart';
 import 'package:flame/events.dart';
 import 'package:flame/experimental.dart';
 import 'package:flame/game.dart';
+import 'package:flame_test/flame_test.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  final draggableTester = GameTester(_GameWithHasDraggableComponents.new);
+  final dualDraggableTester = GameTester(_GameWithDualDraggableComponents.new);
+
   group('HasDraggableComponents', () {
-    testWidgets(
```

Comment:
I don't think these changes to the tests are actually needed. They don't reduce the boilerplate in any way, plus they make the tests weaker.

When a test runs through `testWidgets()`, I can see how the `game` instance is created, and then put inside a `GameWidget`, and that widget is pumped through the test framework. When there is a "helper" function involved, I'll start to worry that it performs some "sneaky" initialization of the `game` instance (as it did before), and the code wouldn't actually work if it was used by an actual user who has no access to `@internal` methods.

Reply:
Yeah, since there is no need for the initialization hack anymore I can revert these.
But it would be nice to be consistent with how we do the tests, and now `testGameWidget` is used on some places and `testWidgets` is used on some.